### DISCUSSION TITLE
Change the default size of font

### DISCUSSION THREAD

**Discussion by shengkui** [tamlok/vnote#1602] [Q&A] [answered]
The default font size of VNote 3.0 is a bit bigger, and I feel very uncomfortable. I always have to reduce the display ratio (CTRL+-) to make it easier to see.
I haven't found a place to set it yet. It is recommended to support setting the font size in the future.

**Comment by tamlok** [comment, marked as answer]
HI,

It is strange that vnote does not remember the zoom factor and you need to do it every time. May be a bug.

Read mode or edit mode?

You could open the default config folder at the top right corner and go to themes/native folder. Then you could modofy web.css or text-editor.theme.

I will expose a GUI setting for this.

Thanks! BTW, you are the first user to Discussions! :)

**Reply by shengkui** [reply]
I achieved my goal by modifying the CSS, thank you for your help.
